Tests/Functional/RunnerConfigRowsTest.php

```diff
@@ -212,13 +212,10 @@ private function getComponent()
                         ],
                         'commands' => [],
                         'entry_point' => 'mkdir /data/out/tables/mytable.csv.gz && '
-                            . 'chmod 000 /data/out/tables/mytable.csv.gz && '
                             . 'touch /data/out/tables/mytable.csv.gz/part1 && '
                             . 'echo "value1" > /data/out/tables/mytable.csv.gz/part1 && '
```

Comment:
@ondrejhlavacek nebylo by lepsi ten test zduplikovat a dat ho tam jednou s ficurou a jednou bez?

Reply:
no zatr, asi jo no :-(

Reply:
i když mi přijde, že testuje funkčnost, kterou už máme otestovanou jinde (`ContainerTest.php`). chybí test, že se pouští `normalizePermissions` na základě featury, kterej je teda tímhle oklikou suplovanej

Reply:
no ani ne, tohle je funkcionalni test na to, ze opravdu jde spustit kontejner s chmodem a bude fungovat, jako jedna vec je jestli se tam preda parameter --user (to je containerTest), ale tady se testuje jestli to fakt funguje